rules/integrations/endpoint/elastic_endpoint_security.toml

```diff
@@ -25,7 +26,7 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.kind:alert and event.module:(endpoint and not endgame)
+event.kind:alert and event.module:(endpoint and not endgame) and not event.category: behavior
```

Comment:
Should we use 'event.code: behavior'? I think that matches Behavior Protection alerts specifically. I think other alert types might have 'event.category: behavior' and we don't want to match on those

Reply:
fixed